<launch>
    <node pkg="camera1394" type="camera1394_node" name="camera1" output="screen" >
      <rosparam file="$(find cameras_tryphon)/launch/ffmv2c.yaml" />
      <param name="guid" value="00b09d0100da7529" />
      <param name="camera_info_url" value="file://$(find cameras_tryphon)/launch/calibration/camera1.yaml" />
	<param name="use_ros_time" value="true" />
      <remap from="camera" to="camera1" />
    </node>
</launch>
